doc/dev/deduplication.rst

```diff
@@ -149,3 +149,64 @@ Aside from radosgw, completing work on manifest object support in the
 OSD particularly as it relates to snapshots would be the next step for
 rbd and cephfs workloads.
 
+How to use deduplication
+========================
+
+Currently, Ceph provides deduplication using RADOS machinery.
+The below is explanation about how users use deduplication.
+
+
+1. Find dedupable chunks using ceph-dedup-tool. ::
+
```

Comment:
Suggest adding `.. code:: bash` to align with formatting elsewhere in Ceph docs:
>.. code:: bash
>
>    ceph-dedup-tool --op estimate --pool $POOL --chunk-size chunk_size  
>      --chunk-algorithm fixed|fastcdc --fingerprint-algorithm sha1|sha256|sha512